### ISSUE TITLE
Tab/Window Capture API

### ISSUE THREAD

**Issue by schme16** [nwjs/nw.js#2733]
I've been trying to find a way to capture the node-webkit window (just the page); I've looked at the getUserMedia path with chromeMediaSource: 'screen' set, but it captures the entire screen/s. I also stubled upon the chromeMediaSource: 'tab' option, but it seems only to be available to chrome extensions (which are, of course, not compatible at this time).

Going through the code in: node-webkit/src/media/media_stream_devices_controller.cc I can see a reference to: request_.video_type == content::MEDIA_TAB_VIDEO_CAPTURE (Line number: 74). that seems to imply that the underlying api is there....

tl;dr
I'm looking to capture from just the node-webkit window, any ideas?
##
